data_adapter_oemof/mappings/GLOBAL_PARAMETER_MAP.yaml

```diff
@@ -1,8 +1,13 @@
-amount: ammount
-#capacity: installed_capacity
-#marginal_cost: variable_costs
-#fixed_cost: fixed_costs
-#capacity_cost: capital_costs
-#overnight_cost: sedos-overnight_cost
-#lifetime: sedos-lifetime
-#wacc: sedos-wacc
+DEFAULT:
+  amount: ammount
+  #capacity: installed_capacity
+  #marginal_cost: variable_costs
+  #fixed_cost: fixed_costs
+  #capacity_cost: capital_costs
+  #overnight_cost: sedos-overnight_cost
+  #lifetime: sedos-lifetime
+  #wacc: sedos-wacc
+WindOnshore:
```

Comment:
@henhuy Should this be the classname e.g. `VolatileAdapter` or more specific like right now? In this case, the `technology` information needs to be passed by your adapter or come from somewhere else...

Reply:
For now, I think this should be the class name, aka the str you get when you do `cls.__name__`, which is `VolatileAdapter` in the upper case.